### ISSUE TITLE
a11y: Colour contrast of focus state and text on header/footer too low

### ISSUE THREAD

**Issue by dstorey** [smashingmagazine/redesign#105]
http://leaverou.github.io/contrast-ratio/#%236e1711-on-%23e33a2c
http://leaverou.github.io/contrast-ratio/#%238E2017%20-on-%23e33a2c

Color is #6e1711
Background color is : #e33a2c
Focus colour: # 8E2017 

This is really low, and makes it both very difficult to read the text in the footer and the focus state of the text and focus ring. Even for my eyes with good short distance vision. 

This fails WCAG criteria 1.4.3 (level AA) http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html

**Comment by dstorey** [comment]
![lowcontrast](https://cloud.githubusercontent.com/assets/465069/24066887/6e7c2c26-0b33-11e7-87eb-394d74e99222.png)

**Comment by jeankaplansky** [comment]
Still a problem on 11/27/2017. Visited 153, and 186 before I found this one.

**Comment by vitalyfriedman** [comment]
Thank you so much for bringing our attention to it, @jeankaplansky — will look into it first thing tomorrow mornig!

**Comment by jeankaplansky** [comment]
@vitalyfriedman - you're welcome!
